firefox-ios/Client/Frontend/Browser/MainMenu/Views/MainMenuDetailsViewController.swift

```diff
@@ -116,7 +116,8 @@ class MainMenuDetailsViewController: UIViewController,
             store.dispatch(
                 MainMenuAction(
                     windowUUID: self.windowUUID,
-                    actionType: MainMenuDetailsActionType.tapBackToMainMenu
+                    actionType: MainMenuDetailsActionType.tapBackToMainMenu,
+                    currentTabInfo: submenuState.currentTabInfo
```

Comment:
oh is this why you need the current tab info in detail state? hmmmmm. Presumably, this is only for `isHomepage` so we might only need that, then?